src/System.Reflection.Context/src/System.Reflection.Context.csproj

```diff
@@ -8,7 +8,7 @@
     <UseECMAKey Condition="'$(UseECMAKey)' == ''">true</UseECMAKey>
     <AssemblyVersion>4.0.1.0</AssemblyVersion>
     <OutputType>Library</OutputType>
-    <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
+    <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netcore50</PackageTargetFramework> 
```

Comment:
maybe we should change the builds file?  We should only be using the empty targetgroup for dotnet and dnxcore builds.  (The dnxcore builds will later be replaced with dotnet/netstandard builds.)